docs/index.md

```diff
@@ -44,15 +44,15 @@ provider "cyral" {
 
 #### New Credentials
 
-A `Service Account` must be created in order to use the provider. It can be created through the control plane UI, accessing the `Service accounts` section in the left menu and clicking on the `+` button. Choose a name for the new service account and select the following roles so you can use all the provider functions:
+An `API Client Credentials` account must be created in order to use the provider. It can be created through the control plane UI, accessing the `API Client Credentials` option in the left menu and clicking on the `+` button. Choose a name for the new account and select the following roles so you can use all the provider functions:
```

Comment:
> An `API Client Credentials` account 

Is this correct wording? Shouldn't it be "API client credentials must be created ..."?

Reply:
You mean without the `account` word?